Assets/Scripts/CameraController.cs

```diff
@@ -252,4 +260,17 @@ public void SetIdle(bool idle)
     {
         _idle = idle;
     }
+
+    public void ResetCameraAngle()
+    {
+        float yDiff = transform.eulerAngles.y - StartingYRotation;
+        RotateX(-yDiff);
+        RotateX(-currentAutoRotate);
```

Comment:
Seems like the starting rotation is fixed. So for some stages there will the camera will reset in a non-scenic view. If we have time i will work on that.

Reply:
Or maybe it should rotate according to the face direction of the character?

Reply:
the camera will rotate to face the level. Don't know what you mean by scenic, but it's at least practical maybe?

Reply:
btw, I notice how stupid this wording looks X(Y), gonna change it to make it more correct